dynawo/cmake/cpu_count.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```

Comment:
Corresponding code should be deleted from top CMakeLists.txt and be replace by this call

Reply:
I'm not sure I understand. On github I only see one line... You mean maybe I can get rid of old codes ? I will do that when I will be sure the code is not used somewhere else.

Reply:
Yes I mean you copy/paste code in a new file but don't delete the old one present in the top CMakeLists

Reply:
That is a very small part. I may still leave the old code only to be sure not to break any further include. That is why I change names of variables and functions (and why I'm interested in naming in french: to avoid conflicts while WIP). In actual CMakeLists files all variables are global, and since maybe used in files included from files included by... That explains it ?

Reply:
You can change the files as if it was the final version: this MR will not be merged until the full development is ready